### DISCUSSION TITLE
Old documentation

### DISCUSSION THREAD

**Discussion by f-cramer** [ory/hydra#3365] [Q&A]
Hi all
After the release of Hydra 2.0 I can not find the documentation for 1.11 anywhere. Unfortunatelly I am currently not able to update to 2.0 just yet, but I at least want to update from 1.10 to 1.11.

**Comment by vinckr** [comment]
Was there something specific you are looking for? 
Theoretically its all in the git history, also see:
https://github.com/ory/hydra/blob/master/CHANGELOG.md
and
https://www.ory.sh/docs/hydra/self-hosted/upgrade
